hack/bump_versions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+
+metallb_version=$(cat hack/metallb_version.txt)
+
+find . -type f -name "*.yaml" -exec sed -E -i "s/quay\.io\/metallb\/speaker:[a-z0-9\.]+/quay.io\/metallb\/speaker:$metallb_version/g" {} \;
+find . -type f -name "*.yaml" -exec sed -E -i "s/quay\.io\/metallb\/controller:[a-z0-9\.]+/quay.io\/metallb\/controller:$metallb_version/g" {} \;
+
+operator_version=$(cat hack/operator_version.txt)
+find . -type f -name "*.yaml" -exec sed -E -i "s/quay\.io\/metallb\/metallb-operator:[a-z0-9\.]+/quay.io\/metallb\/metallb-operator:$operator_version/g" {} \;
+find . -type f -name "*.yaml" -exec sed -E -i "s/quay\.io\/metallb\/metallb-operator-bundle-index:[a-z0-9\.]+/quay.io\/metallb\/metallb-operator-bundle-index:$operator_version/g" {} \;
```

Comment:
I found yq much cleaner and easier to read vs sed, and u won't need regex, anything prevent using it ? since its already used it in the other scripts ?

Reply:
Not really, and makes sense. Will change it.